Initial commit

mook-as opened the pull request:
This adds the acceptance tests, and required packages.

Most of the functionality regarding configuration isn't tested yet; I've only confirmed that CATS can _start_.

mook-as committed:
Initial commit

This adds the acceptance tests, and required packages.

mook-as commented:
Group configuration is done; we can now limit to specific test groups.  Should be ready for review.

Not sure if the package vendoring via bosh works; might need to copy the sources instead…

f0rmiga commented on jobs/acceptance-tests/templates/run.erb:
```diff
@@ -0,0 +1,9 @@
+set -e
+
+<% if p("bpm.enabled") %>
+exec /var/vcap/jobs/bpm/bin/bpm run smoke_tests
```
This is wrong.

mook-as replied:
Fixed in 294f869e844409006b8de0f79291e1e904ecb3ed ; though I notice that things still magically worked, _somehow_, even though `bpm.enabled` is true…

f0rmiga commented on jobs/acceptance-tests/templates/config.json.erb:
```diff
@@ -0,0 +1,48 @@
+config[:use_existing_organization] = !properties.acceptance_tests.existing_organization.nil?
+config[:artifacts_directory] ||= '/var/vcap/sys/log/acceptance_tests/'
+
+# TODO: do something with p('acceptance_tests.include')
```
Is this TODO still relevant? We are doing _something_ with it below.

mook-as replied:
Fixed (comment removed) in 294f869e844409006b8de0f79291e1e904ecb3ed

mook-as committed:
jobs/acceptance-tests: address review feedback

- Fix running with BPM
- Remove obsolete TODO comment

Addresses:
https://github.com/mook-as/cf-acceptance-tests-release/pull/1#pullrequestreview-290582619
